_sass/icons.scss

```diff
@@ -0,0 +1,149 @@
+.candlestick-icon{
+	background-image: svg-url('<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 50">
```

Comment:
please see the example which Mohammad provided. make sure all follows the same style